tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -101,7 +101,7 @@ PIPELINE_PCM_ADD(sof/pipe-volume-demux-playback.m4,
 
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s24le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
```

Comment:
Looks good, we just need to hide this change behind a flag similar to line [L#254](https://github.com/thesofproject/sof/pull/3951/files#diff-b7f104af155fbccdb4b07c6e979780f55d60f46ab72e4377f2492f178a872986R254)